.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,65 @@
+name: Bug report
+about: Create a report to help us improve
+labels: "Unconfirmed Bug"
```

Comment:
Historically in the past, I've only used Unconfirmed Bug when a developer has not been able to replicate the issue on their end. I'd argue that no label means the same thing here.

Reply:
Maybe a "needs testing" and/or "type: bug" sort of thing?

Reply:
I'll remove it for now because we have to give it a label anyway when we respond, but it can be changed, of course, in the future.

Reply:
(Also we couldn't think of a good name as an alternative)